lib/vsc/filesystem/lustre.py

```diff
@@ -130,6 +130,8 @@ def __init__(self):
         self.filesets = {}
         self.quotadump = '/var/cache/lustre'
 
+        self.quota_types = Typ2Param
```

Comment:
also use the `self.quota_types` in the methods

Reply:
Done in https://github.com/hpcugent/vsc-filesystems/pull/83/commits/4f764b97cd1967265065bf023d3c17e932dcce97